src/parser/jobs/blm/modules/RotationWatchdog.js

```diff
@@ -244,6 +269,20 @@ export default class RotationWatchdog extends Module {
 				</Trans>,
 			}))
 		}
+
+		this.checklist.add(new Rule({
+			name: <Trans id="blm.rotation-watchdog.checklist.dots.name">Keep your <StatusLink {...STATUSES.THUNDER_III} /> DoT up</Trans>,
+			description: <Trans id="blm.rotation-watchdog.checklist.dots.description">
+				Your <StatusLink {...STATUSES.THUNDER_III} /> DoT contributes significantly to your overall damage, both on its own, and from additional <StatusLink {...STATUSES.THUNDERCLOUD} /> procs. Try to keep the DoT applied.
```

Comment:
I'm not 100% happy with the wording here but dunno something better... basically I don't want people to spam t3ps because that's a pps loss. You always try to maximize your dot uptime with the least amount of thunder casts neccessary. To me the suggestion sounds a bit like "casting lots of thunder = good" when it has to be more nuanced